squire_core/src/state/accounts.rs

```diff
@@ -12,11 +13,22 @@ use mongodb::{
 };
 use serde::{Deserialize, Serialize};
 use squire_sdk::{
-    actor::*,
     api::{Credentials, RegForm},
     model::{accounts::SquireAccount, identifiers::SquireAccountId},
 };
+use tokio::sync::oneshot::Sender as OneshotSender;
 use tracing::Level;
+use troupe::{
+    ActorBuilder,
+    ActorState,
+    Permanent,
+    Scheduler,
+    sink::{
+        SinkActor,
+        SinkClient,
+        permanent::Tracker,
+    },
+};
```

Comment:
nit: You should be able to use a wildcard import for this.
```suggestion
use troupe::prelude::*
```

Reply:
I see. I would have thought that wildcard is less preferable because it could clobber names. Though I'm guessing that Rust would tell you if that happened (unlike Python). Do you recommend it generally or just if I'm importing a ton of stuff like this?

Reply:
Opinions on wildcard imports vary. You are correct that Rust will warn you about this (even between different versions of the same crate). But, generally speaking, if a crate has a `prelude` module, this module should be thought of as "tools you need to use this crate" and/or "things you are going to commonly import".

As a general rule, I usually only use wildcard imports on prelude modules or something similar.